src/components/RichTextEditor/EditorActions/Plugins/CheckboxListPlugin/CheckboxListElement.tsx

```diff
@@ -1,14 +1,17 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import React from 'react';
+import { ReactEditor } from 'slate-react';
 import { PlateRenderElementProps, TElement, TodoListItemNodeData, getRootProps, setNodes } from '@udecode/plate';
 import { merge } from '@utilities/merge';
-import React, { FC } from 'react';
-import { ReactEditor } from 'slate-react';
+import { MarkupElement } from '../MarkupElement';
+import { ELEMENT_CHECK_ITEM } from './id';
 
-export const CheckboxItemElement: FC<PlateRenderElementProps> = (props) => {
+export const CheckboxListElementTag = (props: PlateRenderElementProps) => {
```

Comment:
Rename to `CheckboxListElement`?

Reply:
I cant because underneath there is a component with this name already:)

Reply:
Got idea, i need to distinguish between the names and I will do that with the postfix NODE => CheckboxListElementNode